src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,392 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using DelimitedFiles: readdlm
+using LinearAlgebra: eigvals
+using .PolynomialOptimizer
+
+import .PolynomialOptimizer: filter_eigvals, undo_normalization!, bisection
+
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level 
+# PERK (Paired-Explicit Runge-Kutta) time integration schemes
+abstract type PERK end
```

Comment:
As noted somewhere else, I am not a huge fan of abbreviations that are not ubiquitous in a given community, as it makes the code harder to understand the code for the uninitiated. I thus recommend something like this, here, and all derivative types:
```suggestion
abstract type AbstractPairedExplicitRungeKutta end
```
Alternatively, if this is too long for the derivative types below, I could also live with `AbstractPairedExplicitRK` :-)

Reply:
Would it be preferable to rename files containing the term 'PERK' as well, given this consideration?